### ISSUE TITLE
User Name not visible while logged in 

### ISSUE THREAD

**Issue by khan01** [salesagility/SuiteCRM#2432]
When logged in as any user, User name should be visible in Top Right corner or either in the dropdown menu since managing multiple users as an admin we quickly need to know which user are we logged in with. This will be handy rather than clicking on profile or employee and going to check which user are we logged in with. 

![suitecrm user name issue](https://cloud.githubusercontent.com/assets/582150/19449690/08708d26-94c0-11e6-8299-825cf14043c9.png)
#### Issue

As an admin of CRM system, we usually need to set different users settings and preferences as we like. The user himself also would like to see what Name is set in the Profile, along with a Profile Image. This was previously part of SugarCRM and had different Profile images and showing user Display names for quickly checking which user is logged in along with profile picture. 

Using SuiteP theme and logging as different user, its very lenght process to find what User display name is set and there is no profile picture.  These were considered a bit handy in terms of taking screenshots and presenting them to various clients as the Profile Name and Image would show a distinction between various roles. 

If this feature can be brought back along which was alreay there in SugarCRM. 
#### Expected Behavior

When user logged in, top right Icon should be replaced with user Profile Image along with user display name.  
#### Actual Behavior

Avatar is used as Profile Image and their is no option to update this. No user name displayed in Top Right Menu. 
#### Possible Fix

Add a File Field to Users module to be used as Profile Pic. 

Display Name should be adjusted in the Top Right Menu.
#### Steps to Reproduce

1.
2.
3.
4.
#### Your Environment

- SuiteCRM Version used: 7.7.4
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): 
- Environment name and version (e.g. MySQL, PHP 7):
- Operating System and version (e.g Ubuntu 16.04):

**Comment by shogunpol** [comment]
The issue has been tested in recent version of SuiteCRM(7.7.5), and not appear,
@khan01 , can you give me some additional info? for example what browser you use, 
what version of PHP  and SQL you use, also operating system
Thanks for your contribution.

**Comment by Mausino** [comment]
@shogunpol i think that he is thinking on this

![screen1](https://cloud.githubusercontent.com/assets/5773777/19477155/0fc3d1d2-953d-11e6-8f5c-9e9dc51036c8.png)

@khan01 If the width resolution of your screen is more as >1560 px you are able see the name not only icon. Here is videoscreen: http://screencast.com/t/pDOUwrohDS

**Comment by khan01** [comment]
@shogunpol On Chrome Version Version 53.0.2785.143 m and Screen Resolution 1280 x 800. 
@Mausino So Only Large screens with resolution 1600 or more are able to see User Name. That can also be adjusted on screens like 1024 or 1280 at least.  Also user profile image rather than Avatar is a must addition. Each system offer its users to have a profile pic. Why Not CRM??

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [User Name not visible while logged in](https://trello.com/c/BiF3TTrR/66-user-name-not-visible-while-logged-in)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
